### PR TITLE
3939 decimal quantities requisition line

### LINKED ISSUE #3939: Decimal quantities in requisition_line
Changing the pack_size to a float size, the requisition quantity column need to be updated as well:

```rust
table! {
    requisition_line (id) {
        id -> Text,
        requisition_id -> Text,
        item_link_id -> Text,
        item_name -> Text,
        requested_quantity -> Integer,
        suggested_quantity -> Integer,
        supply_quantity -> Integer,
        available_stock_on_hand -> Integer ,
        average_monthly_consumption -> Integer,
        snapshot_datetime -> Nullable<Timestamp>,
        approved_quantity -> Integer,
        approval_comment -> Nullable<Text>,
        comment -> Nullable<Text>,
    }
}
```

### PR DESCRIPTION
**Pull request by clemens-msupply**
Fixes #3939 

# 👩🏻‍💻 What does this PR do?

Updates requisition line quantities to be float

# 🧪 Testing
- Run tests
- Create internal order in msupply containing float quantities, the line should get synced
- go through UI and look for related problems

### CODE REVIEW

**Review by roxy-dao — COMMENTED** [review]
Thanks Clemens! I just have one question below (:

File: server/service/src/requisition/request_requisition/add_from_master_list.rs [hunk 1]
```diff
@@ -344,17 +345,16 @@ mod test {
...
-            test_item_stats::item_1_soh() as i32
+            test_item_stats::item_1_soh()
         );
         assert_eq!(
             line.requisition_line_row.average_monthly_consumption,
-            test_item_stats::item1_amc_3_months() as i32
+            test_item_stats::item1_amc_3_months()
         );
-        assert_eq!(
+        assert_approx_eq!(
```
[the remark this answers is not in the document]
Comment (clemens-msupply) [reply server/service/src/requisition/request_requisition/add_from_master_list.rs:354]: Yes this was due to some floating point math issue... (diff < epsilon)

**Review by roxy-dao — APPROVED** [review]
Thanks Clemens~!

File: server/graphql/general/src/queries/tests/item_stats.rs [hunk 2]
```diff
@@ -57,7 +57,7 @@ mod tests {
                     "id": &test_item_stats::item2().id,
                     "stats": {
                         "averageMonthlyConsumption":  test_item_stats::item2_amc_3_months(),
-                        "availableStockOnHand":  test_item_stats::item_2_soh(),
+                        "availableStockOnHand":  test_item_stats::item_2_soh() ,
```
[the remark this answers is not in the document]
Comment (clemens-msupply) [reply server/graphql/general/src/queries/tests/item_stats.rs:60]: thanks! formatter is not working in macro...

**Pull request merged**